src/CONST.js

```diff
@@ -46,6 +48,22 @@ const CONST = {
         RESERVED_FIRST_NAMES: ['Expensify', 'Concierge'],
     },
 
+    CALENDAR_PICKER: {
+        // Numbers were arbitrarily picked.
+        MIN_YEAR: CURRENT_YEAR - 100,
+        MAX_YEAR: CURRENT_YEAR + 100,
+    },
+
+    DATE_BIRTH: {
+        MIN_AGE: 5,
+        MAX_AGE: 150,
+    },
+
+    DIRECTION: {
```

Comment:
Although if we left this generic so it can be applied to any icon, maybe can mention that here - that this is used to be able to apply a rotate / transform style to any component